swarm/src/lib.rs

```diff
@@ -1398,6 +1286,7 @@ where
 
 /// Parameters passed to `poll()`, that the `NetworkBehaviour` has access to.
 // TODO: #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
Which part is dead code? Can we remove it?

Reply:
@thomaseizinger `local_peer_id` and `listened_addrs` are unused now but I think they are not needed because they were used previously by `local_peer_id` and `listened_addresses` functions that are now removed from the codebase. As a consequence, I've just removed these fields.